src/renderer/utils/createStorage.ts

```diff
@@ -1,73 +1,210 @@
-// Helper to work with browser's local/session storage api
+// Helper for work with persistent local storage (default: window.localStorage)
+// TODO: write unit/integration tests
 
-export interface IStorageHelperOptions {
-  addKeyPrefix?: boolean;
-  useSession?: boolean; // use `sessionStorage` instead of `localStorage`
-}
+import { app, remote } from "electron";
+import path from "path";
+import { ensureDirSync, readJsonSync, writeJson } from "fs-extra";
+import type { CreateObservableOptions } from "mobx/lib/api/observable";
+import { action, comparer, observable, reaction, toJS, when } from "mobx";
+import produce, { Draft, setAutoFreeze } from "immer";
+import { isEqual, isFunction, isObject } from "lodash";
+
+setAutoFreeze(false); // allow to merge observables
 
-export function createStorage<T>(key: string, defaultValue?: T, options?: IStorageHelperOptions) {
+export function createStorage<T>(key: string, defaultValue?: T, options?: StorageHelperOptions<T>) {
   return new StorageHelper(key, defaultValue, options);
 }
 
-export class StorageHelper<T> {
-  static keyPrefix = "lens_";
+export interface StorageHelperOptions<T> extends StorageConfiguration<T> {
+  autoInit?: boolean; // default: true
+}
 
-  static defaultOptions: IStorageHelperOptions = {
-    addKeyPrefix: true,
-    useSession: false,
-  };
+export interface StorageConfiguration<T> {
+  storage?: StorageAdapter<T>;
```

Comment:
Given that the default storage adaptor in this PR doesn't cover all `T` this field should be made required. (I don't think that is a breaking change since we don't export `StorageHelper`).